web/gds-user-ui/src/components/ReviewSubmit/index.tsx

```diff
@@ -71,6 +72,11 @@ const ReviewSubmit: React.FC<ReviewSubmitProps> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isSent]);
 
+  // Update the review step status to complete if the user submits a registration.
+  if (isTestNetSubmitted || isMainNetSubmitted) {
```

Comment:
The stepper label color doesn't immediately update after the registration is submitted. To fix this, I think `react-query` will have to be used which would require some refactoring that should be handled in a separate story.